### PR TITLE
BMD-4599 drone.yml prepared

### PR DESCRIPTION
**Pull request by bmxmaza**
### Jira ticket:
https://berlingskemedia.atlassian.net/browse/BMD-4599

### Do not merge
Do not merge until this task will be finished: https://berlingskemedia.atlassian.net/browse/BMD-4881

### CODE REVIEW

File: server/api_consumers/mdb_client.js [hunk 1]
```diff
@@ -11,11 +11,15 @@ try {
     MDBAPI_ADDRESS = process.env.MDBAPI_ADDRESS;
   } else if (process.env.MDBAPI_PORT) {
     MDBAPI_ADDRESS = `http://${process.env.MDBAPI_ADDRESS}:${process.env.MDBAPI_PORT}`
+  } else if(!temp.protocol && process.env.MDBAPI_ADDRESS) {
```
Comment (dgolabek-sii) [inline server/api_consumers/mdb_client.js:14]: space before if (is there linter to run in the project?)

File: server/index.js [hunk 2]
```diff
@@ -41,6 +43,15 @@ const init = async () => {
     }
   });
 
+  // Remove this route when task BDM-5915 will be done
```
Comment (dgolabek-sii) [inline server/index.js:46]: is there task in Jira for that?

**Review by dgolabek-sii — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 2 above)
Comment (bmxmaza) [reply server/index.js:46]: Yes.

File: server/api_consumers/mdb_client.js [hunk 3]
```diff
@@ -11,11 +11,15 @@ try {
     MDBAPI_ADDRESS = process.env.MDBAPI_ADDRESS;
   } else if (process.env.MDBAPI_PORT) {
     MDBAPI_ADDRESS = `http://${process.env.MDBAPI_ADDRESS}:${process.env.MDBAPI_PORT}`
+  } else if (!temp.protocol && process.env.MDBAPI_ADDRESS) {
+    MDBAPI_ADDRESS = `https://${process.env.MDBAPI_ADDRESS}`;
+
```
Comment (dgolabek-sii) [inline server/api_consumers/mdb_client.js:16]: please remove empty line

File: server/api_consumers/mdb_client.js [hunk 4]
```diff
@@ -11,11 +11,15 @@ try {
...
   } else {
     throw new Error();
   }
 } catch (ex) {
-  console.error('Env var MDBAPI_ADDRESS missing or invalid.');
+  console.log(process.env.MDBAPI_ADDRESS);
```
Comment (dgolabek-sii) [inline server/api_consumers/mdb_client.js:21]: was it added for purpose or should it be removed?

**Review by dgolabek-sii — DISMISSED** [review]

**Review by dgolabek-sii — DISMISSED** [review]

**Pull request merged**